.github/workflows/ci.yml

```diff
@@ -83,6 +83,12 @@ jobs:
     - name: List installed Python packages
       run: python -m pip list
 
+    - name: Debug
+      if: matrix.os == 'macos-13' || matrix.os == 'macos-latest'
+      run: |
+        ls -l $(brew --prefix)/lib
+        otool -L $(python -c "import wasserstein; print(wasserstein.__file__.replace('__init__.py', '_wasserstein.cpython-311-darwin.so'))")
```

Comment:
This file isn't present, maybe print out what is present first?

Reply:
Ah, yeah, this was a ChatGPT-inspired Hail Mary that didn't get me anywhere. I meant to revert it. 

My current thought processes is that the errors have something to do with why the following lines were needed in [setup.py](https://github.com/thaler-lab/Wasserstein/blob/master/setup.py#L71C4-L75C49)

```
if platform.system() == 'Darwin':
        omp_cxxflags.insert(0, '-Xpreprocessor')
        libs = ['omp']
        ldflags = ['-Wl,-rpath,/usr/local/lib', '-L/opt/homebrew/lib', '-Wl,-rpath,/opt/homebrew/lib']
        includes.append('/opt/homebrew/include')
```

Reply:
Let's turn on verbose logging, so we can see what's happening. The OpenMP support in CMake should be doing this already.

Reply:
```diff
- python -m pip install --upgrade ".[test]"
+ 
```python -m pip install --upgraden --verbose -Ccmake.verbose=1 -Clogging.level=INFO ".[test]"

Reply:
I'm very confused how that change (implemented in [this commit](https://github.com/j-s-ashley/Wasserstein/pull/5/commits/5dd6d4e2fd6be958b8000a70674e2fe15fa1f189)) led to [this error](https://github.com/j-s-ashley/Wasserstein/actions/runs/10204619181/job/28233534498?pr=5#step:8:13) on Windows:
```
pytest: D:\a\_temp\ac2d2819-671b-4b1f-b223-5e5a70717972.ps1:2
Line |
   2 |  pytest tests
     |  ~~~~~~
     | The term 'pytest' is not recognized as a name of a cmdlet, function, script file, or executable program. Check
     | the spelling of the name, or if a path was included, verify that the path is correct and try again.
Error: Process completed with exit code 1.
```

Reply:
It’s erroring and continuing: `Arguments to -C must be of the form KEY=VAL`

but… that is what we did…